src/averageAges.js

```diff
@@ -15,11 +15,22 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const maleFiltered = people.filter((person) => person.sex === 'm');
+
+  const centuryFiltered = maleFiltered.filter((person) => {
+    const calculatedCentury = (Math.ceil(person.died / 100));
```

Comment:
Consider using constants with meaningful names instead of magic numbers. It's considered programming antipattern